tracing/shared/primitives/rpc/debug/src/lib.rs

```diff
@@ -55,7 +55,8 @@ sp_api::decl_runtime_apis! {
 			max_priority_fee_per_gas: Option<U256>,
 			nonce: Option<U256>,
 			access_list: Option<Vec<(H160, Vec<H256>)>>,
-		) -> Result<(), sp_runtime::DispatchError>;		
+			authorization_list: Option<AuthorizationList>,
```

Comment:
This may break master, we probably need to add a new feature like `runtime-3900`.